build.sh

```diff
@@ -23,6 +23,8 @@ php composer.phar install
 php artisan migrate --force
 php artisan lang:js resources/assets/js/messages.js
 
+php artisan db:seed --force
```

Comment:
build.sh is run during deployment so this will probably destroy production data if run